language/examples/document-qa/question_answering_documents_langchain_matching_engine.ipynb

```diff
@@ -0,0 +1,1385 @@
+{
```

Comment:
<p>Can remove bullet point at start of sentence or at least make the symbol consistent across bullet point icons used?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>